src/HomeArticles.jsx

```diff
@@ -0,0 +1,19 @@
+import { allArticles } from './Api';
+import { useEffect } from 'react';
+import { SingleArticle } from './SingleArticle';
+
+export const HomeArticles = ({ articles, setArticles }) => {
+	useEffect(() => {
+		allArticles().then((response) => {
+			setArticles(response.articles);
+		});
+	}, [setArticles]);
```

Comment:
This "setArticles" in the useEffect dependancy array isn't doing anything, you could and should take this out and it should work the same. (The "set" functions will never change and therefore should never be in a dependancy array for useEffect.) 🙂